orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/CompoundExecutionOperatorSpec.groovy

```diff
@@ -29,6 +30,10 @@ class CompoundExecutionOperatorSpec extends Specification {
   ExecutionRunner runner = Mock(ExecutionRunner)
   def execution = Mock(Execution)
 
```

Comment:
may be here also we can do setup and cleanup instead of the entire spec ?

Reply:
I don't think this is necessary, the tests in this class don't put anything in the MDC